rachellegelden/Home-Together-Canada
# HTC 559: Create Paginate Component + Styling

# [Issue 559](https://github.com/rachellegelden/Home-Together-Canada/issues/559)

closes #559

## Summary
- Created Paginate Component with Test
- Removed Previous Search Controller
- Updated Containers to implement Paginate 

## Relevant Motivation & Context
- Feature

## Testing Instructions
- Login as member 
- Navigate to Search Members
- Click on numbers below to see other results on next/prev page

## Developer checklist prior to opening this pull request:


## Reviewer
